src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -134,9 +135,9 @@ class Client extends BaseTransport implements QueryTransport, WritingInterface,
     private $checkLoginOnServer = true;
 
     /**
-     * @var array
+     * @var \ArrayObject
      */
-    protected $namespaces = array();
+    protected $namespaces;
```

Comment:
it would be nice to use the NamespaceRegistry here - but we don't have these things available at the transport level.